grpc/src/main/java/com/replica/RouterImpl.java

```diff
@@ -327,8 +327,16 @@ private void augmentLegsForPt(ResponsePath path) {
                 path.getLegs().add(RouterConverters.toCustomWalkLeg(thisLeg, travelSegmentType));
             } else if (leg instanceof Trip.PtLeg) {
                 Trip.PtLeg thisLeg = (Trip.PtLeg) leg;
-                path.getLegs().add(
-                        RouterConverters.toCustomPtLeg(thisLeg, gtfsFeedIdMapping, gtfsLinkMappings, gtfsRouteInfo));
+
+                long startTime = System.currentTimeMillis();
+                CustomPtLeg customPtLeg = RouterConverters.toCustomPtLeg(thisLeg, gtfsFeedIdMapping, gtfsLinkMappings, gtfsRouteInfo);
+                double durationSeconds = (System.currentTimeMillis() - startTime) / 1000.0;
+                if (statsDClient != null) {
+                    statsDClient.histogram("routers.to_custom_pt_leg_seconds", durationSeconds);
+                }
+                logger.info("routers.to_custom_pt_leg_seconds: " + durationSeconds);
```

Comment:
I'm fine doing this on a one-off basis if we can't get the metrics working, but I think we should try it without logs first - one log per PT request could easily get very spammy, and if new metrics aren't working, we should probably try to tackle that (as it blocks us in a lot of other areas too)